build

```diff
@@ -77,7 +41,7 @@ while getopts "f:a:h" option; do
     ACCENTS="$OPTARG"
     ;;
   h)
-    HYPRCURSOR=1
+    echo "uneeded, hyprcursor is generated by default now."
     ;;
```

Comment:
We're not providing an official API or anything so feel free to remove the hyprcursor related code.